example/yaml-json-schema.yaml

```diff
@@ -0,0 +1,100 @@
+$id: https://vendor.example/schemas/credential.yaml
+title: Example Credential using a YAML-based JSON Schema
+description: >-
+  An example verifiable credential with schema.
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/ns/credentials/v2",
+      "https://www.w3.org/ns/credentials/examples/v2"
+    ],
+    "id": "http://vendor.example/credentials/1776",
+    "type": ["VerifiableCredential", "NewCredentialType"],
+    "issuer": {
+      "id": "did:example:123",
+      "type": ["Organization", "OrganizationType"]
+    },
+    "validFrom": "2010-01-01T19:23:24Z",
+    "credentialSchema": {
+      "id": "https://vendor.example/schemas/credential.yaml",
+      "type": "JsonSchema"
+    },
+    "credentialSubject": {
+      "id": "did:example:456",
+      "type": ["Person", "JobType"],
+      "claimName": "claimValue"
+    }
+  }
```

Comment:
This example isn't a VC, since it isn't secured. I would suggest either adding `proof` to make it a `vc+ld+json`, or making it a `vc+ld+json+jwt`.

Reply:
I agree.

Reply:
what about updating to `an example unsecured verifiable credential`

Reply:
See comment here: https://github.com/w3c/vc-json-schema/pull/200#discussion_r1289155661

I suggest working the who thing backwords from the verifiers perspective, and assuming the verify will reject anything that is not secured, or dereferenced from a trusted source.

Reply:
maybe this should be updated to just show a json schema in yaml format